lib/form.js

```diff
@@ -33,7 +33,7 @@ module.exports = function(req, opts){
       if (err) return done(err);
 
       try {
-        done(null, qs.parse(str));
+        done(null, qs.parse(str, opts.queryString));
```

Comment:
Furthermore, should `qs` be renamed, or replaced, the option would not be dependant of the dependency name.

On another thought, even the JSON options should be defined as it's proper key; `json`. It could still be BC by merging the options, etc., but this goes out of this scope, here.